<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>37</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>37</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SkypeBot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/ico.ico</normaloff>:/icon/ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>317</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Управление</string>
    </property>
    <addaction name="aConnectToSkype"/>
    <addaction name="aDisconnectSkype"/>
    <addaction name="separator"/>
    <addaction name="aStartSkype"/>
    <addaction name="aExitSkype"/>
    <addaction name="separator"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Словарь</string>
    </property>
    <addaction name="aRefreshDictionary"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Чаты</string>
    </property>
    <addaction name="aDialogChats"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="aStartSkype">
   <property name="text">
    <string>Запустить скайп</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="aExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aExitSkype">
   <property name="text">
    <string>Выход из скайпа</string>
   </property>
  </action>
  <action name="aConnectToSkype">
   <property name="text">
    <string>Соединится со скайпом</string>
   </property>
  </action>
  <action name="aDisconnectSkype">
   <property name="text">
    <string>Отсоединиться от скайпа</string>
   </property>
  </action>
  <action name="aRefreshDictionary">
   <property name="text">
    <string>Перечитать</string>
   </property>
  </action>
  <action name="aDialogChats">
   <property name="text">
    <string>Список чатов</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
